### PR TITLE
Feature/ga

### PR DESCRIPTION
**Pull request by zenln**
Adding ga http://hpc.pnl.gov/globalarrays/

### CODE REVIEW

File: var/spack/repos/builtin/packages/ga/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    homepage = "http://hpc.pnl.gov/globalarrays/"
+    url = "https://github.com/GlobalArrays/ga/archive/v5.6.1.tar.gz"
+
+    version('5.6.1', 'bf63d6b4170eefa8548bcbd46912989b')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('openmpi')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/ga/package.py]: can it not be build with `mpich` or `intel-mpi`?

Comment (zenln) [inline var/spack/repos/builtin/packages/ga/package.py]: Yes, I think it can be done.

Comment (davydden) [inline var/spack/repos/builtin/packages/ga/package.py]: then write `depends_on('mpi')`

Comment (zenln) [inline var/spack/repos/builtin/packages/ga/package.py]: Oh Ok thanks.

**Comment by zenln** [comment]
Thanks. Will just use that.

**Pull request closed without merging**